modules/test/playwright/pages/journal-content-web/WebContentDisplayPage.ts

```diff
@@ -162,14 +181,27 @@ export class WebContentDisplayPage {
 
 		if (webContentName) {
 			await this.selectWebContentInConfigurationFrame
-				.getByText(webContentName)
+				.getByText(webContentName, {exact: true})
 				.waitFor({state: 'visible'});
 			await this.selectWebContentInConfigurationFrame
-				.getByText(webContentName)
+				.getByText(webContentName, {exact: true})
 				.hover();
 			await this.selectWebContentInConfigurationFrame
-				.getByText(webContentName)
+				.getByText(webContentName, {exact: true})
 				.click();
+
+			if (waitAfterAddingWebcontent) {
```

Comment:
Can we rethink this to avoid using `waitAfterAddingWebcontent`? We could use `.waitFor()` to wait for something to be in the screen 🤔